roles/mongodb/vars/main.yml

```diff
@@ -2,3 +2,4 @@
 public_ipaddr: "{{ hostvars[inventory_hostname][ 'ansible_' + primary_if ].ipv4.address }}"
 internal_ipaddr: "{{ hostvars[inventory_hostname][ 'ansible_' + private_if ].ipv4.address }}"
 mongodb_bind_host: "{{ internal_ipaddr }}"
+mongo_replSet: "{% for node in groups['controller'] %}{% if hostvars[node]['inventory_hostname'] not hostvars[groups['controller'][0]]['inventory_hostname'] %} {{hostvars[node]['inventory_hostname']}}{% endif %}{% endfor %}"
```

Comment:
Same as above.  The problem being that `inventory_hostname` is not in the dictionary.

See example dictionary items from the following command.
`ansible -m setup 172.17.16.70 -u root`